README.md

```diff
@@ -1,5 +1,47 @@
 A RESTful API for querying the ORACC database using ElasticSearch.
 
+## ElasticSearch management
+We are currently using ElasticSearch 6.
```

Comment:
Checking again, the build server has ES 5. It's also worth keeping in mind that [newer versions of ES require newer versions of `elasticsearch-dsl`](https://elasticsearch-dsl.readthedocs.io/en/latest/#compatibility), although there is no difference for ES5 or 6.
I don't know if we want to make this statement looser ("we are focusing/have tested this on ES6") or just try with ES6 everywhere.